double_linked_list.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+
+class ListItem(object):
+    def __init__(self, val, prev_item=None, next_item=None):
+        self.val = val
+        self.next_item = next_item
+        self.prev_item = prev_item
+
+    def __str__(self):
+        return str(self.data)
+
+
+class DoubleLinked(object):
+    def __init__(self, first_item=None, last_item=None):
+        self.first_item = first_item
+        self.last_item = last_item
+
+    def insert(self, val):
+        # adds val to beginning of list
```

Comment:
Docstrings should be triple-quoted, not [octothorped](http://en.wiktionary.org/wiki/octothorpe)